common/src/main/java/com/microsoft/hydralab/common/util/Const.java

```diff
@@ -73,6 +73,9 @@ interface SmartTestConfig {
         String REQUIRE_FILE_NAME = "requirements.txt";
         String STRING_FOLDER_NAME = "SmartTestString";
         String STRING_FILE_NAMES = "strings,username,password";
+        String XCTEST_ZIP_FILE_NAME = "Xctest.zip";
+        String XCTEST_ZIP_FOLDER_NAME = "Xctest";
+        String XCTEST_RESULT_FILE_NAME = "XctestResult";
```

Comment:
SmartTestConfig interface is used for smarttest. Maybe you can add an interface?

Reply:
Resolved